packages/components/src/Combobox/components/ComboboxContent/ComboboxContent.tsx

```diff
@@ -67,39 +69,43 @@ export function ComboboxContent(props: ComboboxContentProps): JSX.Element {
         />
 
         {searchValue && (
-          <div
-            className={styles.triggerIcon}
-            onClick={() => setSearchValue("")}
+          <button
+            className={styles.clearSearch}
+            onClick={clearSearch}
             data-testid="ATL-Combobox-Content-Search-Clear"
             aria-label="Clear search"
-            role="button"
           >
             <Icon name="remove" size="small" />
-          </div>
+          </button>
         )}
       </div>
+      <div className={styles.list}>
+        {filteredOptions.map(option => (
+          <button
+            className={classnames(
+              styles.option,
+              option.id === selectedOption?.id && styles.selectedOption,
+            )}
+            key={option.id}
+            onClick={() => handleSelection(option)}
+          >
+            {option.label}
+          </button>
+        ))}
 
-      {filteredOptions.map(option => (
-        <button
-          className={classnames(
-            styles.option,
-            option.id === selectedOption?.id && styles.selectedOption,
-          )}
-          key={option.id}
-          onClick={() => handleSelection(option)}
-        >
-          {option.label}
-        </button>
-      ))}
-
-      {filteredOptions.length === 0 && <p>No results for {searchValue}</p>}
-
-      {props.children && <div className={styles.actions}>{props.children}</div>}
+        {filteredOptions.length === 0 && <p>No results for {searchValue}</p>}
+      </div>
+      <div className={styles.actions}>{props.children && props.children}</div>
```

Comment:
not sure if I want to stick with this

the reason I'm keeping the wrapping div even if there were no actions is because that's where I put the `actions:before` to have the gradient

I'm quite sure there's a way to do the same thing with the new `.list` div I added to wrap the options. very open to not rendering the `.actions` and doing it another way!

Reply:
We're going to have some pretty sticky merge conflicts haha - I had to update this quite a bit in my [Actions PR](https://github.com/GetJobber/atlantis/pull/1519/files#diff-3dc00b67c89a4fd17c5f5e14c8a5f794ee1b676a3453f7a29316513f38b86b31) to adjust for styling on the last Action.  It's going to be worth going back after everything is in the feature branch and combing through it once more